### PR TITLE
test - fix fetchLeverageTiers

### PR DESCRIPTION
**Pull request by ttodua**

### CODE REVIEW

File: js/test/Exchange/test.fetchLeverageTiers.js [hunk 1]
```diff
@@ -12,6 +12,13 @@ module.exports = async (exchange, symbol) => {
     //     ],
     // };
     if (exchange.has[method]) {
+
+        const market = exchange.market (symbol);
+        if (market.spot) {
+            console.log (method + '() is not supported for spot market symbol');
```
Comment (carlosmiei) [inline js/test/Exchange/test.fetchLeverageTiers.js:18]: @ttodua "for spot market symbol" seems a bit strange, did you mean "for spot market" + symbol or smth like that?

Comment (samgermain) [reply js/test/Exchange/test.fetchLeverageTiers.js:18]: Maybe it should just be "for spot markets"

Comment (ttodua) [reply js/test/Exchange/test.fetchLeverageTiers.js]: I've just copied that line from https://github.com/ccxt/ccxt/blob/master/js/test/Exchange/test.fetchFundingRateHistory.js#L25

and yes, you are right, we can amend that. (edit: i've pushed changes for both files, please approve whenever you can).

Comment (carlosmiei) [reply js/test/Exchange/test.fetchLeverageTiers.js]: @ttodua nitpick sorry, market should be `markets`

**Comment by carlosmiei** [comment]
LGTM ✅

**Review by carlosmiei — APPROVED** [review]

**Pull request merged**